redirects.csv

```diff
@@ -12,6 +12,10 @@ https://embed.crossroads.net/*,${env:CRDS_ROCK_DOMAIN}mygiving/,302!
 /h,/,302!
 /annualreport/2020,https://annual-report.crossroads.net/,301!
 /annualreports,${env:CRDS_UNIFIED_DOMAIN}annualreports,200!
+/watch,/content-unavailable/,"302! Country=IN,NP"
+${env:CRDS_UNIFIED_DOMAIN}watch,/content-unavailable/,"302! Country=IN,NP"
```

Comment:
These will not do anything on crds-net. Since it's not really possible that if someone was to navigate to crd-unified.netlify.app/watch that they would ever hit the redirects on this repo.

Reply:
When I say these I mean line 16 and 18